guides/user/canary/config/index.md

```diff
@@ -0,0 +1,169 @@
+Spinnaker might be set up so that all configurations are available to all
+pipelines.
+
+1. Hover over the __Delivery__ tab, and select __Canary__.
+
+   ![Select __Canary__ from the __Delivery__ menu.](/guides/user/canary/config/delivery_menu_canary.png)
+1. Select __Add configuration__.
+
+1. Provide a __Name__ and __Description__.
+
+   This is the name shown in the stage config when you create a canary stage for your
+   pipeline.
+
+   ![Canary config declaration](/guides/user/canary/config/canary_config_create.png)
+
+The sections below describe how to specify the metrics and the scoring thresholds
+and weights.
+
+## Create metric groups and add metrics
+
+The metrics available depend on the telemetry provider you use. Spinnaker
+currently supports Stackdriver only.
+
+Metrics are evaluated even if they're not added to groups, but if you want to
+apply the weighting that determines the relative importance of different metrics,
+you need to [add them to groups](#create-metric-groups-and-add-metrics).
+
+1. In the __Metrics__ section, select __Add Metric__.
+
+1. Select the group to add this metric to.
+
+   Data is evaluated for all metrics, but metrics scores only affect the canary
+   evaluation for metrics that are grouped here.
+
+   Click __Add Group__ to create each group you'll use. For example, you might
+   create a group called  "cpu" and add a set of CPU-related metrics. Then,
+   when you configure a new metric here, you would select "cpu" for the
+   CPU-related metrics you add.
+
+1. Give the metric a name.
+
+1. Specify whether this metric fails when the value deviates too high or too low
+compared to the baseline.
+
+   Or select __either__, in which case it fails on deviation in either direction.
+
+1. Optionally, choose a [filter
+template](/guides/user/canary/config/filter_templates/).
+
+   This is only available  if your Spinnaker is [configured for it](). Filter
+   templates are collections of [Stackdriver monitoring
+   filters](https://cloud.google.com/monitoring/api/v3/filters).
+
+   Here's an example:
+
+   ```
+   resource.type = "gce_instance" AND
+   resource.labels.zone = starts_with("${zone}")
+   ```
+
+1. Identify the specific metric you're including in the analysis configuration:
+
+   * In the __Metric Type__ field type at least 3 characters to populate the
+   field with available metrics.
+
+     For example, if you type `cpu` you'll get a list of metrics avaialble from
+     your telemetry provider, from which you can choose.
+
+     ![List of available metrics](/guides/user/canary/config/metric_type_list_cpu.png)
+
+1. Optionally, click __Group by__ and enter the metric metadata attribute by
+which to group and aggregate the data.
+
+   If you're using Stackdriver, you can [group metrics by resource
+   groups](https://cloud.google.com/monitoring/groups/). For example, when you
+   create a metric you can choose to group it by Region. Stackdriver then
```

Comment:
You have to be specific here, since "location" won't be a standard resource label key for a Stackdriver metric, but the example should probably be "zone" instead of "region". 

I think it's more accurate to say something like "You can group a metric's time series by resource or metric label, for example, 'zone' or 'instance_id'."

Reply:
e.g., if you wanted to group by the resource label "zone", you would enter "resource.zone".

Reply:
again this is specific to Stackdriver, and the example is specific to Stackdriver (even though Prometheus also has a notion of "Group By")

Reply:
Ah, right - the linked resource is talking about a different kind of stackdriver group. I can't find the appropriate docs right now, but we should look for something like "group by" aggregation.

Reply:
That linked page is definitely not the thing to link to here. Can find info about `groupBy` here (https://developers.google.com/apis-explorer/#search/timeseries/m/monitoring/v3/monitoring.projects.timeSeries.list) and here (https://developers.google.com/resources/api-libraries/documentation/monitoring/v3/java/latest/com/google/api/services/monitoring/v3/Monitoring.Projects.TimeSeries.List.html#setAggregationGroupByFields(java.util.List)).